docs/designers-developers/developers/tutorials/notices/README.md

```diff
@@ -2,13 +2,13 @@
 
 Notices are informational UI displayed near the top of admin pages. WordPress core, themes, and plugins all use notices to indicate the result of an action, or to draw the user's attention to necessary information.
 
-In the Classic Editor, notices hooked onto the `admin_notices` action can render whatever HTML they'd like. In the Block Editor, notices are restricted to a more formal API.
+In the classic editor, notices hooked onto the `admin_notices` action can render whatever HTML they'd like. In the Block Editor, notices are restricted to a more formal API.
 
-## Notices in the Classic Editor
+## Notices in the classic editor
```

Comment:
Isn't this a title so okay to keep as it is, in order to be consistent with the other titles?

Reply:
Thanks, @torres126 that reminds me I have to push a PR for lowercasing the Block Editor to conform to the core spelling Best Practices;
“block editor” or “block-based editor” | “Block Editor” or “Gutenberg” | When referring to the new editor.
Reference - https://make.wordpress.org/core/handbook/best-practices/spelling/

As the block editor or block-based editor is an interface and not a proper noun it would always be lowercase except when the first word is capitalized due to the beginning of a sentence. This holds true with the classic editor interface, where it's an exception is the Classic Editor plugin as that's the proper name of the plugin.

Reply:
I don't hold a strong opinion on this, but considering it's a title/heading in the document, regardless of whether or not it's capitalised elsewhere, it should still be capitalised in my opinion.

For example, this should otherwise be "Learn more" 

![screenshot_20190303-131659](https://user-images.githubusercontent.com/43215253/53695706-cd58c580-3db6-11e9-844c-ae3eb09ef499.jpg)

And this would be "Displaying notices from your plugin or theme" 

![screenshot_20190303-131716](https://user-images.githubusercontent.com/43215253/53695707-d34ea680-3db6-11e9-93f7-3a9e9aa1e48e.jpg)

The spelling document doesn't mention this scenario, so it's not a big deal at all. :)

Reply:
Thanks @torres126 I see your point there and for consistency have reverted this change and will do the same in #14205 for headings as the current heading style in Gutenberg seems to be capitalize all words. Appreciate the input